src/reports/reports-menu.md

```diff
@@ -4,7 +4,7 @@ title: Reports Menu
 
 Magento provides a wide selection of reports to keep you informed on your marketing efforts, sales products, and customer activity. The Reports menu provides easy access to current information about your sales, products, customers, and promotions.
 
-![]({% link images/images-ee/admin-menu-reports-ee.png %}){: .zoom}
+![]({% link images/images-ee/admin-menu-reports.png %}){: .zoom}
```

Comment:
```suggestion
![Reports Menu]({% link images/images-ee/admin-menu-reports.png %}){: .zoom}
```

Reply:
Adding text inside the square brackets (`[Reports Menu]`) gives the image an alt tag for better accessibility.